decompile.bat

```diff
@@ -1,15 +1,15 @@
 @echo off
 
-set "AMONGUS=D:\Steam\steamapps\common\Among Us"
+gamepath.bat
 
 mkdir tmp
 cd tmp
 
-Il2CppInspector-cli.exe -i "%AMONGUS%\GameAssembly.dll" -m "%AMONGUS%\Among Us_Data\il2cpp_data\Metadata\global-metadata.dat" -h "cpp" --cpp-compiler MSVC
+..\Il2CppInspector-cli.exe -i "%AMONGUS%\GameAssembly.dll" -m "%AMONGUS%\Among Us_Data\il2cpp_data\Metadata\global-metadata.dat" -h "cpp" --cpp-compiler MSVC

```

Comment:
First of all, thank you very much for your work!

This assumes The install location of `Il2CppInspector-cli.exe`, however usually I would expect people to have it in their PATH. A good place to define it would be `gamepath.bat` I think.

I will merge your patch and adjust it.